ansible/roles/atmo-common/tasks/main.yml

```diff
@@ -59,6 +59,11 @@
   tags:
     - timezone
 
+- name: Restart rsyslog so that logs are written with correct timezone
+  service:
+    name: rsyslog
+    state: restarted
```

Comment:
is `rsyslog` a service that will potentially not be installed on an OS?
perhaps a `when:` clause would be good here to ensure we don't crash on this task?